normandy/control/static/control/js/utils/formHelpers.js

```diff
@@ -0,0 +1,23 @@
+/* Redux forms with complex/nested data structures require
+ * a specialized fields array to handle objects and arrays.
+ * Eg: ['surveyId', 'surveys[].title', 'defaults.thankYouMessage']
+ */
+const reduxFormFields = {
```

Comment:
I think it'd be clearer to store these in the form files instead of in a shared helpers file.